common/djangoapps/third_party_auth/tests/specs/base.py

```diff
@@ -272,6 +272,21 @@ def get_request_and_strategy(self, auth_entry=None, redirect_uri=None):
 
         return request, strategy
 
+    def _get_login_post_request(self, strategy):
+        """Gets the login post request given a strategy with a completed pipeline."""
+        request = self.request_factory.post(reverse('login_api'))
+
+        # Note: It doesn't seem exactly right to copy all this from strategy.request, but
+        # this is more correct than the earlier version of this code that used strategy.request
+        # for login, rather than a POST request to login_api.
```

Comment:
Nit: how useful do you think comments are that refer to past code?  In order for a developer to understand and see the value of this comment, they are required to dig through history.  Is there another variant of this comment that allows one to understand the rationale behind the new code without requiring extra effort?